lib/ui/painting.dart

```diff
@@ -1591,6 +1602,11 @@ enum ImageByteFormat {
   /// image may use a single 8-bit channel for each pixel.
   rawUnmodified,
 
+  /// Raw extended range RGBA format.
+  ///
+  /// Unencoded bytes, in RGBA row-primary form with straight alpha, 32 bit float per channel.
```

Comment:
does "32 bit float" mean IEEE 754  `binary32`?

Reply:
might be worth mentioning the Dart classes you can use to decode this (I assume `Float32List` would work?).

Maybe even include some sample code!

Reply:
Done, added sample code and specified the float32 format